src/main/java/seedu/address/model/company/Company.java

```diff
@@ -130,4 +135,13 @@ public String toString() {
                 .toString();
     }
 
+    /**
+     * Returns a string representation of the company in the form of a list of attributes.
+     * Used to display the entire string representation in one line
+     */
+    public String styledOutput() {
+        return "Name=" + name + "," + " Role=" + role + "," + " Status=" + status + ","
+                + " Deadline=" + deadline + "," + " Recruiter Name=" + recruiterName + ","
+                + " Phone=" + phone + "," + " Email=" + email + "," + " Priority=" + priority;
+    }
```

Comment:
Remove fields that are not necessary to be displayed to user, when they see a duplicate company error message.